app/models/preneeds/date_range.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+require 'common/models/form'
+
+module Preneeds
+  class DateRange < Preneeds::Base
+    attribute :from, String
+    attribute :to, String
+
+    def self.permitted_params
+      attribute_set.map { |a| a.name.to_sym }
```

Comment:
since DateRange inherits from Preneeds::Base which might define its own attributes, is it worth being explicit about what these attributes are? I think there is a risk here with not being explicit.

Reply:
So you mean like
```
def self.permitted_params
 [:from, :to]
end
```

Reply:
yes